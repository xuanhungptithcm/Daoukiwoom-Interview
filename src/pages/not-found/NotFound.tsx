import { Link } from "react-router-dom";
import { PRIVATE_PATH } from "../../components/routing/constants";
import "./not-found.scss";
const NotFound = () => {
  return (
    <div className="not-found">
      <div className="image">
        <img src="assets/icons/jpg/404.jpg" alt="" />
      </div>
      <div className="title">Oops!</div>
      <div className="description">
        We canâ€™t seem to find the page you are looking for
      </div>
      <div className="help-link">
        <Link to={PRIVATE_PATH.DASHBOARD}>Go back to home</Link>
      </div>
    </div>
  );
};

export default NotFound;
