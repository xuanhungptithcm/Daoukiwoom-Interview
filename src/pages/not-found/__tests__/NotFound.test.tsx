/**
 * @jest-environment jsdom
 */

import { shallow } from "enzyme";
import NotFound from "../NotFound";

describe("NotFound", () => {
  test("should render correct", () => {
    expect(shallow(<NotFound />)).toMatchSnapshot();
  });

});
