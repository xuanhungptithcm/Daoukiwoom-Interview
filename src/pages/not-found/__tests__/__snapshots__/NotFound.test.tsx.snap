// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should render correct 1`] = `
<Fragment>
  404 Page not found
</Fragment>
`;
